lib/Service/ParticipantService.php

```diff
@@ -961,10 +961,10 @@ public function changeInCall(Room $room, Participant $participant, int $flags, b
 			$flags &= ~Participant::FLAG_WITH_VIDEO;
 		}
 
-		if ($flags !== Participant::FLAG_DISCONNECTED) {
+		if ($flags !== Participant::FLAG_DISCONNECTED && !$silent) {
```

Comment:
We still need the Events to trigger so the HPB notifies users about the participant, etc.
So extend the event with the parameter and inside the listener for the call notification return early when the silent parameter is true